src/authentication.ts

```diff
@@ -172,7 +172,7 @@ export async function getUser(
     throw new ExpiredTokenError();
   }
 
-  if (!isValidScopes(token, scopes)) {
+  if (!isValidScopes(token, permissios)) {
```

Comment:
@markusahlstrand it's going to blow up here

```suggestion
  if (!isValidScopes(token, permissions)) {
```